### ISSUE TITLE
Controller crashing

### ISSUE THREAD

**Issue by PattyzinhaCis** [refinedmods/refinedstorage#3737]
### Describe the bug

I was playing my modpack and I did things for the storage, and when I went to put the controller down to start making the terminal for my items, my game just crashed, so I logged in again, and tried to put it in again, when it crashed again and then I realized that it was the controller that was having a problem.

### How can we reproduce this bug or crash?

1. make a controller and put it on the floor

### What Minecraft version is this happening on?

Minecraft 1.18.2

### What Forge version is this happening on?

Forge 47.3.0

### What Refined Storage version is this happening on?

1.12.4

**Comment by Nodrance** [comment]
![We need logs, please include them](https://github.com/user-attachments/assets/dbabc354-4242-4280-b653-8f69c381330d)

**Comment by Nodrance** [comment]
![We need logs, please provide them](https://github.com/user-attachments/assets/7d120cf9-89ee-4bc5-b97f-83974ed3a5f1)

**Comment by Boolyman** [comment]
My server is crashing when I put the controller down as well.  Here is the latest crash report.

https://pastebin.com/FqKL1mpt


Edit:
I got the block placed, and it crashed.  So when the server came back up, I broke the block and it crashed again.  It was not adjacent to any other blocks.  
I have since switched the mod out for Applied Energistics, so I won't be back to follow up on this ticket, but figured the log would be helpful for you to help others.
